ts/packages/node/src/stores/ORM.ts

```diff
@@ -1,59 +1,72 @@
 import 'reflect-metadata';
 import { ConfigStore, type ConfigStoreQuery, type Config as IConfig } from '@configu/ts';
-import { Entity, Column, DataSource, type DataSourceOptions, Index, PrimaryGeneratedColumn } from 'typeorm';
+import { DataSource, type DataSourceOptions, Entity, Index, PrimaryGeneratedColumn, Column } from 'typeorm';
 import _ from 'lodash';
 
-@Entity()
-@Index(['set', 'key'], { unique: true })
-class Config {
-  @PrimaryGeneratedColumn('uuid')
-  id: string;
+const createEntity = (tableName: string) => {
+  @Entity({ name: tableName })
+  @Index(['set', 'key'], { unique: true })
+  class Config {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
 
-  @Index('set')
-  @Column('text')
-  set: string;
+    @Index(`${tableName}_set`)
```

Comment:
I'm unsure about the meaning of this change, please explain.

Reply:
Index name is unique across the entire db. it has nothing to do with the actual indexing

Reply:
Each table should have an index on the set column, why should the index name be unique? There's no restriction that I'm aware of.

Reply:
As discussed, let's just give ORM the responsibility for naming the index. Please mention this as a note on the PR for future reference.